packages/now-cli/src/util/input/input-project.ts

```diff
@@ -69,9 +69,17 @@ export default async function inputProject(
       });
       const projectName = answers.existingProjectName as string;
 
-      const loader = wait('Verifying project name…', 1000);
-      project = await getProjectByIdOrName(client, projectName, org.id);
-      loader();
+      if (!projectName) {
+        output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```

Comment:
Don't we have a util for printing errors?

Reply:
Yea, `output.error` should work the same way

Reply:
Good point!

```suggestion
        output.error(`Project name cannot be empty`);
```